src/intgtests/test_pam_hbac.py

```diff
@@ -333,6 +337,7 @@ def _driver_setup(self):
                                               admin_password,
                                               bind_dn,
                                               bind_pw,
+                                              start_ssl = True,
```

Comment:
PEP8 complains:
src/intgtests/test_pam_hbac.py:340:56: E251 unexpected spaces around keyword / parameter equals
src/intgtests/test_pam_hbac.py:340:58: E251 unexpected spaces around keyword / parameter equals

I think it would be nice to use same style as for 'insecure',